spec/spec_helper.rb

```diff
@@ -16,15 +16,87 @@
 
 module Minitest
   class Test
+    extend T::Sig
     include ContentHelper
     include TemplateHelper
 
     Minitest::Test.make_my_diffs_pretty!
-    def indented(str, indent)
-      str.lines.map! do |line|
-        next line if line.chomp.empty?
-        " " * indent + line
-      end.join
+  end
+end
+
+class DslSpec < Minitest::Spec
+  before(:all) do
```

Comment:
Can we add a comment about what is happening here please?

Reply:
Done!